types/are.na/index.d.ts

```diff
@@ -0,0 +1,621 @@
+// Type definitions for are.na 0.1
+// Project: https://github.com/ivangreene/arena-js
+// Definitions by: Hunter Caron <https://github.com/huntercaron>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.1
+
+/**
+ * Timestamp in ISO 8601 format, eg. "2020-06-20T15:59:04.796Z"
+ */
+type Timestamp = string;
+
+/**
+ * Parameter type for API functions
+ */
+interface Params {
+    [key: string]: any;
+}
+
+/**
+ * Parameter type for API functions with pagination support
+ */
+interface PaginationParams extends Params {
+    page: number;
+    per: number;
+}
+
+/**
+ * Arena class, organized hierarchically as nested objects
+ */
+declare class Arena {
+    /**
+     * Arena class, organized hierarchically as nested objects
+     *
+     * ```js
+     * // Request for a channel with slug of "arena-influences"
+     * const arena = new Arena();
+     * arena.channel("arena-influences").get()
+     * ```
+     * @param accessToken - Your are.na API access token
+     * @param baseURL - Base URL to make requests on (default: https://api.are.na/v2/)
+     */
+    constructor(config?: { accessToken?: string; baseURL?: string });
```

Comment:
missing `authToken` and `requestHandler` 
https://github.com/ivangreene/arena-js/blob/master/index.js#L32
https://github.com/ivangreene/arena-js/blob/master/index.js#L40